pkgs/os-specific/linux/mbp-modules/mbp2018-bridge-drv/default.nix

```diff
@@ -33,6 +33,6 @@ stdenv.mkDerivation rec {
     license = lib.licenses.gpl2Only;
     platforms = platforms.linux;
     maintainers = [ lib.maintainers.hlolli ];
-    broken = kernel.kernelOlder "5.4";
+    broken = kernel != null && (kernel.kernelOlder "5.4" || (kernel.isCachy && (lib.versionAtLeast kernel.version "6.10")));
```

Comment:
why do we need kernel != null here?

Reply:
Some of the kernel modules had this, I guess it isn't really necessary